pkg/extensions/search/schema.graphql

```diff
@@ -96,12 +59,16 @@ type RepoSummary {
      Vendors: [String]
      Score: Int
      NewestTag: ImageSummary
```

Comment:
Since the field returns an ImageSummary, perhaps a better name would be NewestImage